app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt

```diff
@@ -192,6 +193,14 @@ class UseCaseModule {
     fun provideObserveConversationListDetailsUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId) =
         coreLogic.getSessionScope(currentAccount).conversations.observeConversationListDetails
 
+    @ViewModelScoped
+    @Provides
+    fun observeObserveUserListByIdUseCaseProvide(
```

Comment:
Shouldn't it be `provideObserveUserListByIdUseCase`? 😄

Reply:
will change it 😅